docker/sandbox/flyte-entrypoint-dind.sh

```diff
@@ -60,5 +66,8 @@ helm upgrade -n flyte --create-namespace flyte $charts --kubeconfig /etc/rancher
 
 wait-for-flyte.sh
 
+# With flytectl sandbox --source flag, we mount the root volume to user source dir that will create helm & k8s specific directory. It cause issue while doing fast register 
+rm -rf /root/.cache /root/.kube /root/.config
+
```

Comment:
Not your change , but any plan to combine the two file -dind and default or have a common code refactored so that we dont have duplicate such fixes later on

Reply:
it's issue with docker-in-docker